src/explore-education-statistics-common/src/modules/charts/components/ChartRenderer.tsx

```diff
@@ -52,11 +52,41 @@ function ChartRenderer({
   const chart = useMemo(() => {
     switch (props.type) {
       case 'line':
-        return <LineChartBlock {...props} />;
+        return (
+          <>
+            {props.axes === undefined ||
+            props.axes.major === undefined ||
+            props.axes.minor === undefined ? (
+              <div>Unable to render chart, chart incorrectly configured</div>
+            ) : (
```

Comment:
Thinking about it, this type of error handling / validation isn't great as it's not particularly robust and doesn't actually allow us to track these types of issues properly in Application Insights.

Would suggest that we just ditch this stuff completely and just let any errors bubble up so that they can be logged at the nearest error boundary.

Same applies for other cases below.

Reply:
Yep, makes sense. I've removed these and the tests for it.